golint zero exit code

**Describe the bug**

`lint-golint` stage in `Makefile` does not return error code on non empty warnings after processing.

**To Reproduce**

1. Remove comment from one of exported type in source code
2. ```make lint-golint```

**Expected behavior**

As the result `make` must fail and return non zero exit code.

```Found 1 lint suggestions; failing.
make[1]: *** [lint-golint] Error 1
make: *** [lint] Error 2
```

**Environment:**

 - Version: v1.0.0
 - GOOS: darwin, linux
 
**Additional context**

 In `golint` documentation there is a flag `--set_exit_status`, which must solve problem.

--- a/Makefile
+++ b/Makefile
@@ -68,6 +68,7 @@ lint-golint:
 		-v ${PWD}:/project \
 		-w /project \
 		golint:custom \
+			--set_exit_status \
 			/project/pkg/... \
 			/project/internal/... \
 			/project/cmd/...

